scripts/generate_umgap_tables.sh

```diff
@@ -156,6 +156,7 @@ download_and_process_uniprot_tryptic() {
   		--peptide-max "$peptide_max_length" \
       --taxa "$(luz "$output_dir/taxons.tsv.lz4")" \
       --peptides "$(lz "$temp_dir/$temp_constant/peptides-out.tsv.lz4")" \
+      --proteomes "$(lz "$temp_dir/$temp_constant/proteomes.tsv.lz4")" \
```

Comment:
We don't need the proteomes for umgap, right? So, I think we can just remove this here.